src/sentry/snuba/metrics/utils.py

```diff
@@ -189,6 +209,15 @@
 REQUIRES_RHS_CONDITION_RESOLUTION = {"transform_null_to_unparameterized"}
 
 
+def get_entity_keys_of_use_case_id(use_case_id: UseCaseID) -> set[EntityKey] | None:
+    """
+    Returns a set of entity keys that are available for the use_case_id.
+
+    In case the use case id doesn't have known entities, the function will return `None`.
+    """
+    return USE_CASE_ID_TO_ENTITY_KEYS.get(use_case_id)
```

Comment:
Abstracted behind the function to hide a behavior that might change in the future.